### ISSUE TITLE
Train loss and Validation loss are very high!!

### ISSUE THREAD

**Issue by Jasonsun1993** [georgesung/ssd_tensorflow_traffic_sign_detection#34]
I trained the whole LISA dataset, but when the train code was running. I found that the Train loss and Validation loss were up to 2600! Does anybody know what the problem is ??

**Comment by xinxueying** [comment]
I have the same problem. The trainning loss at beginning is over 800, and so as the validation loss. And after about 20000 round training, it drop to about 200.

**Comment by xinxueying** [comment]
I did some edit to some api, so it can train under tensorflow1.4.1

**Comment by Jasonsun1993** [comment]
@xinxueying After training, did you test the images in samples_images folder? Did it have good results?

**Comment by xinxueying** [comment]
@Jasonsun1993 
Problem solved， I changed the learning rate of training to 0.5 (original code use the default 0.01,which is very small considering training from scratch):
OPT = tf.train.AdadeltaOptimizer()
change to 
OPT = tf.train.AdadeltaOptimizer(learning_rate=0.5)

**Comment by Jasonsun1993** [comment]
@xinxueying Really? Did you trained the whole data set?? How about the results?? Besides, can I have your Wechat number? I want to discuss about this code further.

**Comment by zengxyu** [comment]
Have you got the dataset? I can't open the link  LISA Traffic Sign Dataset(http://cvrr.ucsd.edu/LISA/lisa-traffic-sign-dataset.html). If you got the dataset, could you please send it to me?

**Comment by pageedward** [comment]
@xinxueying what is the val loss when u ended training?
